File: benchmark/examples/java-test-suite/calculator/src/src/main/java/cerberus/Calculator.java
package cerberus;

public class Calculator {
	public static int wrongAdd(int a, int b) {
		int result = a + b - 1;
		return result;
	}

	public static int add(int a, int b) {
		int result = a + b;
		return result;
	}

	public static int addThree(int a) {
		int result = a + 3;
		return result;
	}

	public static int addFive(int a) {
		int result = a + 5;
		return result;
	}
}

File: benchmark/examples/python-test-suite/calculator/src/calculator.py
class Calculator:

    def add_wrong(self, x: int, y: int) -> int:
        result = x + y - 2
        return result

    def add(self, x: int, y: int) -> int:
        result = x + y
        return result

    def add_three(self, x: int) -> int:
        result = x + 3
        return result

    def add_five(self, x: int) -> int:
        result = x + 5
        return result
